.vscode/launch.json

```diff
@@ -242,7 +242,9 @@
             "args": [
                 "--extensionDevelopmentPath=${workspaceFolder}/workspaces/ballerina/ballerina-extension",
                 "--extensionTestsPath=${workspaceFolder}/workspaces/ballerina/ballerina-extension/out/test",
-                "${workspaceFolder}/workspaces/ballerina/ballerina-extension/test/data/bi_empty"
+                "${workspaceFolder}/workspaces/ballerina/ballerina-extension/test/data/bi_empty_project",
```

Comment:
Probably need to think of a way to avoid doing this everytime we add a new project